### PR TITLE
Refactor comparer serialization

### PR DESCRIPTION
**Pull request by asi1024**
~Depends on #438.~

### CODE REVIEW

**Comment by asi1024** [comment]
Rebased. PTAL.

File: pytorch_pfn_extras/utils/comparer.py [hunk 1]
```diff
@@ -252,15 +261,15 @@ def compare(self, loaders, n_iters=None):
             for future in concurrent.futures.as_completed(futures):
                 future.result()
 
-    def _add_target(self, handle, models, outputs):
+    def _get_target(self, handle, models, outputs):
         raise NotImplementedError('Comparers must override _add_target')
```
Comment (kmaehashi) [inline pytorch_pfn_extras/utils/comparer.py:265]:
```suggestion
        raise NotImplementedError('Comparers must override _get_target')
```

**Review by kmaehashi — COMMENTED** [review]
LGTM except for nits!

**Comment by asi1024** [comment]
@kmaehashi Fixed. PTAL!

**Review by kmaehashi — APPROVED** [review]

**Pull request merged**